### ISSUE TITLE
How to link and run package locally?

### ISSUE THREAD

**Issue by MichaelBergquistSuarez** [apollographql/apollo-link#507]
I'm trying to use `apollo-link-http` with this branch: https://github.com/apollographql/apollo-link/pull/489 since I'd like to try this package out. Figured I'd run locally until the branch has been merged.

But `npm link` doesn't work from either the root or `packages/apollo-link-http`.

And I can't find anything about how to do this under the `Contributing` section in the readme.
When running `npm link` from the root:
```
npm ERR! Package must have a name field to be linked
```

When running `npm link` from `/packages/apollo-link-http`:
```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for apollo-link-http-common@^0.2.0
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'apollo-link-http'
npm ERR! notarget
```

**Comment by littletower** [comment]
hi, did you manage to make it work?
I really need that fix 😞 major blocker for us...

**Comment by MichaelBergquistSuarez** [comment]
No I didn't. I gave up and simply modified the code directly in `node_moduels/apollo-link/lib/index.js` instead

**Comment by dejayc** [comment]
This is merged into `master` now?

**Comment by littletower** [comment]
`v1.2.0` fixed this issue

**Comment by MichaelBergquistSuarez** [comment]
That's great! It would be neat if there were some explanation about how to link anyway though. For contribution and future similar problems:)

**Comment by evans** [comment]
@MichaelBergquistSuarez Totally agree! Could you write up how to `npm link` a specific package and PR it into the [Contributing section](https://github.com/apollographql/apollo-link#contributing) of the readme?

Since this was fixed with `apollo-link@1.2`, I'm going to close

**Comment by MichaelBergquistSuarez** [comment]
Sorry, I would if I could! But I don't know how to `npm link` the packages in this repo. I'm not too familiar with Lerna
